config.js

```diff
@@ -0,0 +1,10 @@
+export const firebaseConfig = {
+  apiKey: "AIzaSyAcNVmn2nvssOJTVOHoT4NorZ3mScZ4Vgk",
```

Comment:
We usually do not commit API keys to public repo, and add them in .env file. But since these credential are temporary we let share these maybe.